multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/MicrometerConfiguration.java

```diff
@@ -76,42 +49,20 @@ public Duration step() {
                 return Duration.ofSeconds(stepInSeconds);
             }
 
-            @Override
-            public int batchSize() {
-                if (batchSize == null) {
-                    return DynatraceConfig.super.batchSize();
-                }
-                return batchSize;
-            }
-
             @Override
             @Nullable
             public String get(String k) {
                 return null;
             }
         };
-        DynatraceMeterRegistry registry = new DynatraceMeterRegistry(dynatraceConfig, Clock.SYSTEM);
+        JmxMeterRegistry registry = new JmxMeterRegistry(jmxConfig, Clock.SYSTEM);
         registry.config()
                 .meterFilter(MeterFilter.acceptNameStartsWith(CLIENT_CONNECTIONS_METRICS_PREFIX))
                 .meterFilter(MeterFilter.deny());
         Metrics.globalRegistry.add(registry);
         return registry;
     }
 
-    private String getDynatraceToken(DynatraceServiceInfo serviceInfo) {
-        if(serviceInfo == null) {
-            return null;
-        }
-        return serviceInfo.getMicrometerToken();
-    }
-
-    private String getDynatraceUrl(DynatraceServiceInfo serviceInfo) {
-        if (serviceInfo == null) {
-            return null;
-        }
-        return serviceInfo.getMicrometerUrl();
-    }
-
     private DynatraceServiceInfo getDynatraceServiceInfo() {
```

Comment:
Do we need this now?

Reply:
It is not mandatory to check if DynatraceServiceInfo exists before registering JmxMeterRegistry.
However, it we check that we can optimize the things as not registering the registry at all.
In this way we will avoid unecesarry preparation of our own metrics and send them to micrometer